packages/hub/jest/test-setup.ts

```diff
@@ -1,6 +1,6 @@
-import "../config/env";
-import {GanacheServer} from "@statechannels/devtools";
-import {deploy} from "../deployment/deploy";
+import '../env';
```

Comment:
Do we enforce single or double quotes in linting? I couldn't find a linting file for the hub package.

Reply:
https://github.com/statechannels/monorepo/blob/f12dd05d24329f17f3be82b828bfdc878b1f6268/packages/hub/.eslintrc.js#L102

Reply:
Ah there we go - `eslintrc.js`

Reply:
Hmm, is the linting not enforced because this file is outside of `src`?

Reply:
(since the linter was passing happily with double quotes in the imports before this PR)